pages/_app.tsx

```diff
@@ -1,10 +1,11 @@
 import type { AppProps } from 'next/app'
 import Navbar from '@components/NavBar'
 import '@styles/main.css'
-
+import MobileMenu from '@components/MobileMenu/index'
```

Comment:
You don't have to write `index`
```suggestion
import MobileMenu from '@components/MobileMenu'
```